beacon-chain/blockchain/execution_engine.go

```diff
@@ -216,16 +216,30 @@ func (s *Service) notifyNewPayload(ctx context.Context, preStateVersion int,
 	}
 
 	var lastValidHash []byte
-	if blk.Version() >= version.Deneb {
+	switch {
```

Comment:
Why a switch instead of an if-else here?

Reply:
i was just being consistent with other parts of the code, I switched it back now, not sure if the readability is improved